### PR TITLE
Front meals index celia

### PR DESCRIPTION
**Pull request by OlivierCalmels**
# Added Buton for menus validation

### CODE REVIEW

File: app/controllers/recipes_controller.rb [hunk 1]
```diff
@@ -13,6 +13,14 @@ def show
     @doses = Dose.where(recipe_id: params[:id])
     @recipe = Recipe.find(params[:id])
     @meals = Meal.find(params[:id])
+    @restrictions = []
+    @doses.each do |dose|
+      if dose.ingredient_id.list_of_restrictions
+      @restrictions << dose.ingredient_id.list_of_restrictions
+      end
+    end
+    raise
```
Comment (OlivierCalmels) [inline app/controllers/recipes_controller.rb]: Raise?

File: app/controllers/recipes_controller.rb [hunk 2]
```diff
@@ -13,6 +13,14 @@ def show
...
+    @restrictions
```
Comment (OlivierCalmels) [inline app/controllers/recipes_controller.rb]: not used

File: app/views/recipes/show.html.erb [hunk 3]
```diff
@@ -76,6 +76,9 @@
 
   <ul class="recipe-show-list">
 
+  <h4><strong>Restrictions: </strong></h4>
```
Comment (OlivierCalmels) [inline app/views/recipes/show.html.erb]: comment

**Review by shakshouka — APPROVED** [review]
Oh yeaaaaah
tenor-206264666

**Pull request merged**